app/assets/javascripts/components/import_ratings/_modal.html

```diff
@@ -130,9 +178,12 @@ <h2>CSV</h2>
     <span ng-show="ctrl.ratingsTypePicked()">
     This operation WILL override your existing ratings. Proceed with caution!
     </span>
-    <span ng-show="!ctrl.ratingsTypePicked()">
+    <span ng-show="ctrl.informationNeedsTypePicked()">
     This operation WILL override your existing information needs. Proceed with caution!
     </span>
+    <span ng-show="ctrl.snapshotsTypePicked()">
```

Comment:
so, do we override?   or do we just create missing queries?

Reply:
good question!